projects/frontend/shared-components/gui/package.json

```diff
@@ -2,8 +2,8 @@
   "name": "shared-wrapper",
   "version": "0.0.0",
   "engines": {
-    "node": "^14.15.0 || >=16.10.0",
-    "npm": ">=8.5.5"
+    "node": "^14.15.0 || >=18.12.0",
+    "npm": ">=9.7.2"
```

Comment:
Is it necessary to change the npm version as well?

Reply:
We build and ship with a Docker image using this specific version, so it should also be reflected here. One of the core principles when we were open-sourcing the CI was that building locally and in CI should be the same.